src/main/java/br/com/openenade/api/modalidade/ModalidadeController.java

```diff
@@ -0,0 +1,20 @@
+package br.com.openenade.api.modalidade;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
Use @CrossOrigin("*") to allow the application to accept requests from any origin, like @javanlacerda is using in the PR #4